src/components/Calendar.tsx

```diff
@@ -20,11 +20,6 @@ import Button from "./Button";
 import { isSameDay, endOfDay, startOfDay } from "date-fns";
 
 class Calendar extends React.PureComponent<CalendarProps, CalendarState> {
-  static defaultProps: Partial<CalendarProps> = {
```

Comment:
I don't get the rationale behind this.
The PR title probable seems for Controls.ts
Please elaborate.